lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -226,14 +227,17 @@ export abstract class BaseManagedIdentitySource {
 
     public getManagedIdentityUserAssignedIdQueryParameterKey(
         managedIdentityIdType: ManagedIdentityIdType,
-        imds?: boolean
+        imds?: boolean,
```

Comment:
The parameter is named imds (true/false), but in practice this code handles multiple MSI sources (App Service, Azure Arc, etc.), not just IMDS. Because of that, imds can be misleading

Reply:
also `credential` source will be 2023 version I think